README.md

```diff
@@ -7,33 +7,35 @@
 
 SunPy is an open-source Python library for solar physics data analysis.
 
-[![Visit our IRC channel](https://kiwiirc.com/buttons/chat.freenode.com/sunpy.png)](https://kiwiirc.com/client/chat.freenode.com/#sunpy)
+Visit our matrix channel: [#sunpy-general:matrix.org](https://riot.im/app/#/room/#sunpy-general:matrix.org)
 
 Installation
 ------------
 
-To begin, install the following requirements:
+The recommended way to install SunPy is
+with [conda](https://www.continuum.io/downloads). To install SunPy once conda is
+installed run the following two commands:
 
- * [Python](http://www.python.org) (2.7.x, 3.4.x or later)
- * [Astropy](http://astropy.org) (1.0.0)
- * [NumPy](http://numpy.scipy.org/)
- * [SciPy](http://www.scipy.org/)
- * [Matplotlib](http://matplotlib.sourceforge.net/) (1.1+)
- * [suds-jurko](https://bitbucket.org/jurko/suds)
- * [pandas](http://pandas.pydata.org/) (0.10.0+)
- * [beautifulsoup4](http://www.crummy.com/software/BeautifulSoup/)
- * [sqlalchemy](http://www.sqlalchemy.org/)
+    $ conda config --append channels conda-forge
+    $ conda install sunpy
 
-Next, use git to grab the latest version of SunPy:
 
-    git clone https://github.com/sunpy/sunpy.git
-    cd sunpy
-    python setup.py install
+If you want to develop SunPy you will need to install from git, the best way to
```

Comment:
If you want to develop SunPy you will need to install using git.  The best way to